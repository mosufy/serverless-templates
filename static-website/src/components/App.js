import React from "react";

import "assets/styles/App.css";

import HomeHeader from "components/modules/HomeHeader";

function App() {
  return (
    <div className="App">
      <HomeHeader />
    </div>
  );
}

export default App;
