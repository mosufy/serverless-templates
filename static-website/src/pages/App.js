import React from 'react';

import 'assets/styles/App.css';

import Home from './Home';

function App() {
  return (
    <div className="App">
      <Home />
    </div>
  );
}

export default App;
