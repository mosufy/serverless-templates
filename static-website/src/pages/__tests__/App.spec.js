import React from 'react';
import { shallow } from 'enzyme';

import App from '../App';

describe('test App main', () => {
  it('should render correctly', () => {
    const component = shallow(<App />);
    expect(component).toMatchSnapshot();
  });
});
