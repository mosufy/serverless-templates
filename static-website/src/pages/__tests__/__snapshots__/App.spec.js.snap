// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test App main should render correctly 1`] = `
<div
  className="App"
>
  <Home />
</div>
`;
